src/components/AwaitingPayment.vue

```diff
@@ -17,37 +17,41 @@ export default {
 </script>
 
 <template>
-  <div class="text-center bg-amber-400 text-black">
-    <span>Waiting for Payment...</span>
+  <div>
+    <h4 class="text-gray-200 font-bold py-4">Scan QR code with Lightning Wallet</h4>
   </div>
-  <div class="container mx-auto text-center">
-    <div class="columns-auto">
-      <h4 class="text-gray-200 font-bold py-4">Scan QR code with Lighting Wallet</h4>
+  <div class="justify-center text-center">
+    <img v-if="qrCodeUrl" class="payment-qr-code mx-auto py-5" :src="qrCodeUrl" />
+    <div v-else class="mx-auto w-10 py-5">
+      <LoadingSpinner />
     </div>
-    <div class="columns-auto justify-center text-center">
-      <img v-if="qrCodeUrl" class="payment-qr-code mx-auto py-5" :src="qrCodeUrl" />
-      <div v-else class="mx-auto w-10 py-5">
-        <LoadingSpinner />
-      </div>
-    </div>
-    <div class="columns-auto">
-      <div class="inline-flex items-center w-100">
-        <h5 class="text-gray-200 font-bold py-4">Tap QR Code to copy address</h5>
-        <ClipboardDocumentIcon class="mx-2 size-6 text-yellow-500" />
-      </div>
-    </div>
-    <div class="columns-auto py-4 mx-4">
-      <button
-        class="bg-yellow-500 hover:bg-yellow-600 text-black font-bold py-2 px-4 rounded w-full max-w-90"
-      >
-        Open Wallet
-      </button>
+  </div>
+  <div>
+    <div class="inline-flex items-center w-100">
+      <h5 class="text-gray-200 font-bold py-4">Tap QR Code to copy address</h5> <!-- TODO: do this -->
+      <ClipboardDocumentIcon class="mx-2 size-6 text-yellow-500" />
     </div>
   </div>
+  <div class="py-4 mx-4">
+    <button class="open-wallet-btn py-2 px-4 rounded w-3/4">
```

Comment:
note to self: needs to be wired up (add to todo list)

Reply:
I think this is already there? At least as part of this:
`Add in the UI to change Wallet type`

Reply:
My understanding:

Open wallet: is an <a> tag with a url that is a deep link to the chosen wallet, so it opens the relevant wallet app. In the case of lightning, that means the url should be `lightning:lnbc_rest_of_lightning_invoice_here`. For luno/valr/binance/other it will be a custom URL scheme returned by the server.